### DISCUSSION TITLE
data transfer too slow

### DISCUSSION THREAD

**Discussion by back0893** [filecoin-project/lotus#7585] [General Q&A]
hi,guys
I need some progame. I find my lotus client send data too slow,so i find why.
I find in `go-data-transfer` package `channels_fsm.go` ,often record DataQueued.
i use `lotus net bandwidth`,show me.
```
Segment  TotalIn  TotalOut  RateIn  RateOut
Total    9.4 MB   16 GB     0 B/s   0 B/s
```
so,i send 16GB in 2 day.
i also run `lotus-miner net bandwidth`
```
Segment  TotalIn  TotalOut  RateIn   RateOut
Total    40 GB    770 MB    722 B/s  1.5 kB/s
```

**Comment by de-authority** [comment]
It doesn't seem like a program issue.  do you ever change to another storage provider?

**Reply by LexLuthr** [reply]
@back0893 Was this problem resoled? Did you figure out what was the reason for slow data transfer?

**Reply by LexLuthr** [reply]
I am archiving this discussion as there has been no new reply in last 7 days.
